pages/index/index.js

```diff
@@ -52,6 +55,62 @@ const GettingStarted = glamorous(Button)(
   }),
 )
 
+const StyledLiveProvider = glamorous(LiveProvider)({
+  margin: '0 auto',
+  display: 'flex',
+  flexDirection: 'column',
+  flexWrap: 'wrap',
+  justifyContent: 'space-around',
+  maxWidth: '50rem',
+})
+
+const StyledLiveEditor = glamorous(LiveEditor)({})
+
+const StyledLiveError = glamorous(LiveError)((props, {colors, fonts}) => ({
+  color: colors.code,
+  fontFamily: fonts.monospace,
+  backgroundColor: colors.white,
+  flexBasis: '100%',
+  width: '100%',
+  maxWidth: '100%',
+  padding: '1rem',
+}))
+
+
+const HomepageLivePreview = ({ className, ...rest }, { live: { element: Button }, live }) => {
+  return (
+    <glamorous.Div textAlign="center" marginBottom="30px">
+      <Button href="https://github.com/paypal/glamorous" primary>
+        GitHub
+      </Button>
+      <Button href="http://kcd.im/glamorous-help">
+        Try it
+      </Button>
+    </glamorous.Div>
+  )
+}
+
+HomepageLivePreview.contextTypes = LiveContextTypes
+
+
+const StyledLivePreview = glamorous(HomepageLivePreview)({
+  padding: '1rem',
+})
+
+
+function CodePreview({noInline = true, code, scope = {glamorous}}) {
```

Comment:
This is better! Now we can change the other `CodePreview` without effecting this one because I think it makes sense that the `<StyledLivePreview />` is above the `<StyledLiveEditor />` on the home page, but I'd like them to be side-by-side everywhere else.